### PR TITLE
ROS2 DOC: Extend webots documentation

### PR DESCRIPTION
**Pull request by nhjschulz**
* Add webots subfolder with deployment
* Add reference to Webots ROS2 Supervisor
* some spelling improvements

### CODE REVIEW

File: doc/ROS2/README.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ micro-ROS supports a variety of external build system beyond the micro_ros_setup
 # ROS 2 in DCS
 This section describes the integration of micro-ROS in the existing workspace, extending it by implementing a ROS 2 node and the micro-ROS stack. The goal is to connect the ZumoComSystem with the DDS network using DDS-XRCE, making the robot controllable via ROS messages to mimic turtle graphics drawing using the robot.
 
+Webots integeration is discussed on the 
+[Webots subpage](./webots/webots.md).
```
Comment (gabryelreyes) [inline doc/ROS2/README.md:52]: Please add the missing subtitles to the TOC.

**Review by gabryelreyes — CHANGES_REQUESTED** [review]

File: doc/ROS2/uml/turtle_sim_mqtt.plantuml
Comment (nhjschulz) [inline doc/ROS2/uml/turtle_sim_mqtt.plantuml:1]: wrong file, use webots instead of mqtt

File: doc/ROS2/README.md (hunk 1 above)
Comment (nhjschulz) [reply doc/ROS2/README.md]: fixed

File: doc/ROS2/uml/turtle_sim_mqtt.plantuml
Comment (nhjschulz) [reply doc/ROS2/uml/turtle_sim_mqtt.plantuml:1]: fixed

**Review by gabryelreyes — APPROVED** [review]

**Pull request merged**